src/lib/update.js

```diff
@@ -0,0 +1,44 @@
+export const updateEntities = (update, entities, transformed) => {
+    // If update is not supplied, then no change to entities will be made
+
+    return Object.keys(update || {}).reduce(
+        (accum, key) => {
+            accum[key] = update[key]((entities || {})[key], (transformed || {})[key]);
+
+            return accum;
+        },
+        {}
+    );
+};
+
+export const optimisticUpdateEntities = (optimisticUpdate, entities) => {
+    return Object.keys(optimisticUpdate).reduce(
+        (accum, key) => {
+            if (optimisticUpdate[key]) {
```

Comment:
This isn't new code but I don't know why this check is here. Going to try removing it.